docs/jwst/straylight/reference.rst

```diff
@@ -1,20 +1,25 @@
 Reference File
 ==============
-The MIRI MRS straylight correction step uses a straylight mask. There
+The default MIRI MRS stray-light correction step uses Regions reference file. This
+Regions files is the first extension in the MIRI Distortion file and has  been read
+in and pulled into a separate reference type by the assign_wcs step. 
```

Comment:
How can the assign_wcs step be creating new reference files on the fly while the pipeline is running? This whole section on Regions vs Distortion vs assign_wcs creating things is confusing, at least to me.